app/models/search.rb

```diff
@@ -26,6 +26,8 @@ def initialize(params={})
     elsif source == :family
       @scope = Family.includes(:people)
     end
+
+    @ilike = (@scope.connection.adapter_name == "PostgreSQL" ? 'ilike' : 'like')
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.